src/utils/convert.js

```diff
@@ -170,7 +170,7 @@ const renderStep = (type, definition, key) => {
   const templateContext = {
     KEY: snakeCase(key),
     CAMEL: camelCase(key),
-    NOUN: _.capitalize(key),
```

Comment:
We should probably add a default in case noun doesn't exist (it's optional in WB). I think `_.capitalize(key)` is a decent enough default.

Reply:
Fixed in 53cd6d3.